ci/docker/Dockerfile.collect-alma8

```diff
@@ -9,16 +10,19 @@ RUN echo 'http_caching=none' >> /etc/yum.conf && \
     echo 'assumeyes=1' >> /etc/yum.conf && \
     sed -i 's/best=True/best=False/g' /etc/dnf/dnf.conf && \
     dnf install -y dnf-plugins-core
+RUN dnf install -y epel-release
 RUN dnf config-manager --set-enabled powertools
-RUN curl -o centreon-release.rpm "http://yum-1.centreon.com/standard/21.10/el8/stable/noarch/RPMS/centreon-release-21.10-1.el8.noarch.rpm"
-RUN dnf install --nogpgcheck centreon-release.rpm
-RUN dnf config-manager --set-enabled 'centreon-testing*'
+#RUN curl -o centreon-release.rpm "http://yum-1.centreon.com/standard/21.10/el8/stable/noarch/RPMS/centreon-release-21.10-1.el8.noarch.rpm"
+#RUN dnf install --nogpgcheck centreon-release.rpm
+#RUN dnf config-manager --set-enabled 'centreon-testing*'
```

Comment:
Hi @jean-christophe81 , Kindly could you remove these lines.

Reply:
done